tests/test_tracking_grid/results_true.dat

```diff
@@ -0,0 +1,27 @@
+Diagonal track...
+Number of segments = 3
```

Comment:
I mean it depends. I can definitely output that but then a developer looking through the data might not understand the output. We can probably reduce the text in half by doing what you suggest but the output would be far harder to comprehend. We could reduce the output far more by hashing the result, making it even more hard to comprehend and understand though.